clients/fides-js/src/lib/automated-consent.ts

```diff
@@ -84,97 +68,64 @@ export const automaticallyApplyPreferences = async ({
     return false;
   }
 
-  /**
-   * Select the "best" translation that should be used for these saved
-   * preferences based on the currently active locale.
-   *
-   * NOTE: This *feels* a bit weird, and would feel cleaner if this was moved
-   * into the UI components. However, we currently want to keep the GPC
-   * application isolated, so we need to duplicate some of that "best
-   * translation" logic here.
-   */
-  const bestTranslation = selectBestExperienceConfigTranslation(
-    i18n,
-    effectiveExperience.experience_config,
-  );
-  const privacyExperienceConfigHistoryId =
-    bestTranslation?.privacy_experience_config_history_id;
-
   let gpcApplied = false;
   let noticeConsentApplied = false;
   let migratedConsentApplied = false;
 
-  const consentPreferencesToSave = effectiveExperience.privacy_notices.map(
-    (notice) => {
+  const noticeConsentToSave = experience.privacy_notices.reduce(
```

Comment:
can we add a type to this const?

Reply:
It gets inferred as it gets set, so it's not really necessary.